<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    background-color: #ffffff;
    border: 1px solid #d1d1d6;
    border-radius: 3px;
    color: #000000;
	height: 24px;
}

QLineEdit:focus {
    border: 1px solid #007aff;
    background-color: #ffffff;
}

QLineEdit:disabled {
    background-color: #f2f2f7;
    color: #8e8e93;
    border-color: #d1d1d6;
}</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Download path:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="download_path">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="select_path">
           <property name="styleSheet">
            <string notr="true">QPushButton {
    background-color: #ffffff;
    border: 1px solid #d1d1d6;
    border-radius: 5px;
	color: #000000;
    padding: 3px 12px;
}

QPushButton:pressed {
    background-color: #e6e6eb;
}

QPushButton:disabled {
    background-color: #f2f2f7;
    color: #8e8e93;
    border-color: #d1d1d6;
}</string>
           </property>
           <property name="text">
            <string>Choose</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Accounts</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>3</number>
         </property>
         <item>
          <widget class="QPushButton" name="add">
           <property name="styleSheet">
            <string notr="true">QPushButton {
    background-color: #ffffff;
    border: 1px solid #d1d1d6;
    border-radius: 5px;
	color: #000000;
    padding: 3px 12px;
}

QPushButton:pressed {
    background-color: #e6e6eb;
}

QPushButton:disabled {
    background-color: #f2f2f7;
    color: #8e8e93;
    border-color: #d1d1d6;
}</string>
           </property>
           <property name="text">
            <string>Add account</string>
           </property>
           <property name="icon">
            <iconset resource="../resources.qrc">
             <normaloff>:/settings/static/add.png</normaloff>:/settings/static/add.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="acc_type">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <item>
            <property name="text">
             <string>--</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Google</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Yandex</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Dropbox</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="acc_list_view">
         <property name="styleSheet">
          <string notr="true">QTableView {
	background-color: #ffffff;
	border: 1px solid #d1d1d6;
	border-radius: 3px;
	color: #000000;
	selection-background-color: #007aff;
	selection-color: #ffffff;
}</string>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>150</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
